roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -12,6 +12,10 @@ imageConfig:
   latest: {{ openshift_node_image_config_latest }}
 kind: NodeConfig
 kubeletArguments: {{  l2_openshift_node_kubelet_args  | default(None) | lib_utils_to_padded_yaml(level=1) }}
+  kube-api-qps:
```

Comment:
This should be set in default section of `l_openshift_node_kubelet_args`, smth like:
```
l_openshift_node_kubelet_args_default:
  kube-api-qps:
    - "100"
  kube-api-burst:
    - "200"
l_openshift_node_kubelet_args: "{{ openshift_node_kubelet_args | default(l_openshift_node_kubelet_args_default) }}"
```
so that it could be overridden by `openshift_node_kubelet_args`

Reply:
the defaults upstream are burst = 10, qps = 5, this change is 20x larger, and not sure the impact it would have on larger clusters with more nodes.

@jeremyeder

Reply:
Not sure how I missed this, sorry folks.

Looks like your patch recommends a change to defaults and is against 3.10.

@joelsmith are you sure the 3.10 part is necessary?  Can you indicate how you've reproduced? (just asked the same question in the BZ).